etc/terraform-arm64.conf

```
@@ -0,0 +1,33 @@
+# target architecture - i386, amd64 or all
+ARCH="arm64"
+
+# base codename
+BASECODENAME="noble"
+
+# base version
+BASEVERSION="24.04"
+
+# distribution codename
+CODENAME="circe"
+
+# distribution version
+VERSION="8.0"
+
+# distribution channel
+CHANNEL="stable"
```

```suggestion
CHANNEL="daily"
```

Based on etc/terraform-daily-8.0-azure.conf

---

Resolving conversation because this file is arm64 version of `etc/terraform.conf` which is also stable